.github/workflows/dependabot_auto_merge.yml

```diff
@@ -14,12 +14,13 @@ jobs:
         id: metadata
         uses: dependabot/fetch-metadata@v1
         with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          github-token: '${{ secrets.GITHUB_TOKEN }}'
       - name: Add skip changelog label
+        uses: actions-ecosystem/action-add-labels@v1
         if: ${{ steps.metadata.outputs.update-type != 'version-update:semver-major' && steps.metadata.outputs.dependency-type == 'direct:development' }}
-        run: gh pr edit --add-label "Skip Changelog"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          labels: Skip Changelog
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
this is `github_token` whereas the above is `github-token`. Just checking this is the right config to set :)